docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -1272,13 +1272,21 @@
 
                 <tr>
                     <td class="property" id="store-changelog-replication-factor">stores.<span class="store">store-name</span>.changelog.<br>replication.factor</td>
-                    <td class="default">2</td>
+                    <td class="default">stores.changelog.replication.factor.default</td>
```

Comment:
@jwlent55 sorry for the late reply. I would still prefer to have "default" right after the stores. It would be a documentation effort to declare that "default" is a reserved word in Samza configuration (i.e. this would be a pattern we will use for other plural nouns like systems/streams for default as well). I don't think that putting the default at the end will avoid that problem (e.g. what if someone have a store named "changelog"?). Besides, by putting the default as a "scope", we avoided the adhoc addition of default configuration variables and group them into one scope. Thanks!